update-requirements

```diff
@@ -81,29 +103,42 @@ def get_new_content(current_content, requires):
             yield line
 
             if line.startswith(prefix):
-                key = line[len(prefix):]
+                key = strip_prefix(line, prefix)
                 try:
                     for require in requires[key]:
                         yield require
                 except KeyError:
                     raise SystemExit('Unrecognized section found: ' + line)
-                key = '# end package.json {}'.format(key)
+                key = prefix.replace('start', 'end') + key
+
+
+def strip_prefix(line, prefix):
+    return line[len(prefix):]
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Update the nodejs dependencies from package.json')
-    parser.add_argument('packagesfile', help='Path to the packages file',
-                        type=argparse.FileType('r'))
-    parser.add_argument('specfile', help='path to the specfile', default='foreman.spec', nargs='?',
+    parser = argparse.ArgumentParser(description='Update the dependencies within sections')
+    parser.add_argument('requirementsfile', help='Path to the requirements file',
                         type=argparse.FileType('r'))
+    parser.add_argument('specfile', help='path to the specfile', type=argparse.FileType('r'))
+    parser.add_argument('requirementstype', help='the type of conversion',
+                        choices=('npm', 'specfile', 'direct'))
```

Comment:
I would make the usage `update-requirements <requirementstype> <specfile> <requirementsfile>`, not the other way round, but I'd also not block on such a nice :bike: